to_do_application.py

```diff
@@ -17,7 +17,19 @@ def add_task(self):
         print(f"'{usr_inpt}' has been added to the list.")
 
     def remove_task(self):
-        pass
+        usr_inpt = str(input(f"Enter the task to remove {self.tasks}: "))
+        for i in self.tasks:
+            if usr_inpt in self.tasks:
+                self.tasks.remove(usr_inpt)
+                print(f"'{usr_inpt}' has been removed from the list.")
```

Comment:
this validation is perfect, but it should be placed before for loop (line 20-21)